TenancyInformationApi/V1/Infrastructure/UhTenancyAgreement.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Internal;
+
+namespace TenancyInformationApi.V1.Infrastructure
+{
+    [Table("tenagree")]
+    public class UhTenancyAgreement
+    {
+        [StringLength(11)] [Column("tag_ref"), Key] public string TenancyAgreementReference { get; set; }
```

Comment:
What if you format it as :
`[Column("tag_ref")]`
`[Key]`
`[StringLength(11)]`
`public string TenancyAgreementReference { get; set; }`
It may be easier to read

Reply:
Personally I didn't find that easier to read, but if there's a general consensus I'll change to it.